service/build.gradle

```diff
@@ -4,6 +4,9 @@ buildscript {
   }
   dependencies {
     classpath("net.serenity-bdd:serenity-gradle-plugin:2.4.4")
+    classpath("gradle.plugin.au.com.dius.pact.provider:gradle:4.2.3") {
```

Comment:
wanna try test the dep bump as well?

Reply:
couldn't resolve/download the plugin with the point version